2020-02-Spring/0_arrays_matrix/buy_stock/driver.java

```diff
@@ -0,0 +1,29 @@
+public class driver{
+    public static void main(String [] args){
+        int [] arr1 = {3};
```

Comment:
Need to instantiate an object of our solution class in order to call the solution method.

```suggestion
        solution sol = new solution();
        int [] arr1 = {3};
```